src/check-c-linkage-decls.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import sys, os
+
+os.chdir (os.environ.get ('srcdir', os.path.dirname (__file__)))
+
+HBHEADERS = os.environ.get ('HBHEADERS', '')
+if len (HBHEADERS):
+	HBHEADERS = HBHEADERS.split ()
+else:
+	HBHEADERS = [x for x in os.listdir ('.') if x.startswith ('hb') and x.endswith ('.h')]
+
+HBSOURCES = os.environ.get ('HBSOURCES', '')
+if len (HBSOURCES):
+	HBSOURCES = HBSOURCES.split ()
+else:
+	HBSOURCES = [x for x in os.listdir ('.')
+				 if x.startswith ('hb') and (x.endswith ('.cc') or x.endswith ('.hh'))]
+
+fails = 0
+
+for x in HBHEADERS:
+	with open (x, 'r', encoding='utf-8') as f:
+		content = f.read ()
+		if ('HB_BEGIN_DECLS' not in content) or ('HB_END_DECLS' not in content):
```

Comment:
You can de-indent this block to make it a bit more readable.

Reply:
Done, see if now is better, thanks :)